src/components/ScreenshotDialog.vue

```diff
@@ -0,0 +1,49 @@
+<template>
+    <div ref="modal" class="modal fade" tabindex="-1">
+        <div class="modal-dialog modal-xl modal-dialog-centered">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close" />
+                </div>
+                <div class="modal-body"></div>
```

Comment:
Having an image-only modal is not ideal for accessibility.
Please add a title (for example, "Screenshot of the website") to explain what this modal does.

Reply:
Added a Modal title